### PR TITLE
Reader: Keep small images out of the refreshed galleries

### LINKED ISSUE #9159: Gallery Cards: Ignore small images
This post: https://wpcalypso.wordpress.com/read/feeds/34222/posts/1211847530

Creates this card in Reader:
screen shot 2016-11-04 at 11 40 27 am

The gallery card is omitting this big image: https://youarenotsosmart.files.wordpress.com/2016/11/chems.png

But it includes these images:
https://youarenotsosmart.files.wordpress.com/2014/01/screenshot_13.jpg
https://youarenotsosmart.files.wordpress.com/2016/07/screen-shot-2016-07-18-at-11-34-02-am.png
https://youarenotsosmart.files.wordpress.com/2016/08/screen-shot-2016-08-24-at-6-15-48-pm.png
https://youarenotsosmart.files.wordpress.com/2016/07/6a00d83451b36c69e201b8d0ed5719970c-300wi.png

Most of those images are pretty small. 

So, questions:

• Will the gallery card currently pick any image?
• Can we set a minimum size requirement for images to be considered for Gallery cards?
• Will #9149 change the behavior here? 

Ideally we'd ignore any image that's under 350px wide as usual here to avoid little bugs and logos for the gallery card. Possible?

cc @blowery, @bluefuton, @samouri

### PR DESCRIPTION
**Pull request by blowery**
Require a min width of 350px and require enough images to be marked as a gallery card.

Fixes #9159 

To test, compare https://wpcalypso.wordpress.com/read/feeds/34222 to https://calypso.live/read/feeds/34222?branch=fix/reader/9123

### CODE REVIEW

**Comment by blowery** [comment]
before
you_are_not_so_smart_ _reader_ _wordpress_com

after
you_are_not_so_smart_ _reader_ _wordpress_com

**Comment by fraying** [comment]
Much better! 👍

File: client/blocks/reader-post-card/gallery.jsx [hunk 1]
```diff
@@ -2,20 +2,26 @@
...
+import { map, take, filter } from 'lodash';
 
 /**
  * Internal Dependencies
  */
+import { imageIsBigEnoughForGallery } from 'state/reader/posts/normalization-rules';
 import resizeImageUrl from 'lib/resize-image-url';
 
-const GALLERY_ITEM_WIDTH = 206;
+const GALLERY_ITEM_THUMBNAIL_WIDTH = 420;
```
Comment (samouri) [inline client/blocks/reader-post-card/gallery.jsx]: I thought we were gunning for 350, did we change to 420?

**Comment by jancavan** [comment]
Tested and works great 👍

File: client/blocks/reader-post-card/gallery.jsx (hunk 1 above)
Comment (blowery) [inline client/blocks/reader-post-card/gallery.jsx]: this is the width of the thumb, not the width of the image in the gallery itself. Only really matter when there are fewer than 4 images in the gallery. Shouldn't happen though..

**Pull request merged**